src/reporter.cc

```diff
@@ -97,18 +97,9 @@ void SendMetadataRequest(std::vector<json::value>&& entries,
     throw (boost::system::system_error) {
 
   if (entries.size() == 1) {
-    // Add a copy of the single entry, except for the `views` field. This allows
-    // us to sent a request to the batch endpoint when we have a single request.
-    // We're making use of the fact that duplicate metadata is ignored.
-    const json::Object* single_request = entries[0]->As<json::Object>();
-    json::value duplicate_request = json::object({  // ResourceMetadata
-      {"name", single_request->at("name")->Clone()},
-      {"type", single_request->at("type")->Clone()},
-      {"location", single_request->at("location")->Clone()},
-      {"state", single_request->at("state")->Clone()},
-      {"evenTime", single_request->at("evenTime")->Clone()},
-    });
-    entries.emplace_back(std::move(duplicate_request));
+    // Add a copy of the single entry. This allows us to sent a request to the
+    // batch endpoint when we have a single request.
```

Comment:
Am I understanding correctly: We can't call the batch endpoint with a single request? So we send two identical copies of the same request?

Reply:
That is correct. This is a work around, so we don't have to send the single request directly to the publish endpoint.
I have updated the comment - see if this is better.